examples/gpt-2/gpt2_generate_main.py

```diff
@@ -135,42 +78,29 @@ def run_model():
     batch_size = args.batch_size
     max_decoding_length = args.max_decoding_length
 
-    # Load GPT-2 model configuration
-    if args.config_type == "json":
-        gpt2_config = model_utils.transform_gpt2_to_texar_config(
-            args.config_model)
-    elif args.config_type == 'texar':
-        gpt2_config = importlib.import_module(
-            args.config_model)
-    else:
-        raise ValueError('Unknown config_type.')
-
-    assert max_decoding_length <= gpt2_config.position_size, (
+    assert max_decoding_length <= config_model["position_size"], (
         "max_decoding_length should not be greater than position size")
     assert nsamples % batch_size == 0, (
         "nsamples must be dividable by batch_size")
```

Comment:
Also, this assertion here does not make sense -- it is unnecessary for `nsamples` to be divisible (not "dividable") by `batch_size`. I'm guessing this was there because of computation graph issues with TF.